widgets/src/Project/HeaderDetails.jsx

```diff
@@ -49,19 +52,24 @@ return (
             Near.call("social.near", "set", {
               data: { [accountId]: { profile: { name } } },
             }),
+          canEdit: isAdmin
+
         }}
       />
+
       <Widget
         src={`${ownerId}/widget/Inputs.Viewable.OneLiner`}
         props={{
           value: "Simple solutions for complex tasks",
-          id: "tagline",
-          onSave: (tagline) =>
+          id: "one_liner",
+          onSave: (one_liner) =>
             Near.call("social.near", "set", {
-              data: { [accountId]: { profile: { tagline } } },
+              data: { [accountId]: { profile: { one_liner } } },
```

Comment:
This should be `tagline` instead of `one_liner`

Reply:
Done. Thanks